ci_framework/roles/devscripts/tasks/cleanup.yml

```diff
@@ -29,6 +29,14 @@
     chdir: "{{ cifmw_devscripts_repo_dir }}"
     target: clean
 
+- name: Remove the SSH key generated for accessing the platform
```

Comment:
we also want to ensure it's not in the authorized_keys of the user. It's added in the sub_tasks/16_user.yml:
```YAML
 42 - name: Enable password-less access
 43   become: true
 44   ansible.posix.authorized_key:
 45   ┆ user: "{{ cifmw_devscripts_user }}"
 46   ┆ key: "{{ cifmw_devscripts_ssh_key.public_key }}"
 47   ┆ state: present
```

Reply:
Hope the below changes clarify the reason why it wouldn't required.